test/dymaptic.GeoBlazor.Core.Test.Blazor.Shared/Components/WMSLayerTests.razor

```diff
@@ -23,26 +23,30 @@
             Assert.AreEqual(layer?.Id, createdLayer.Id);
             layerViewCreated = true;
         }
-        AddMapRenderFragment(
-            @<SceneView @ref="sceneView"
-                                  class="map-view"
-                                  OnViewRendered="renderHandler"
-                                  OnLayerViewCreate="OnLayerViewCreated">
+
+        AddMapRenderFragment(@<SceneView @ref="sceneView"
+                                         class="map-view"
+                                         OnViewRendered="renderHandler"
+                                         OnLayerViewCreate="OnLayerViewCreated">
                 <Map>
```

Comment:
We should add `<SpatialReference Wkid="102100" />` to this SceneView, because it is giving the "squashed" globe currently, if you pause the test you can see it.